packages/notification-center/src/i18n/languages/zu.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const ZU: ITranslationEntry = {
+  translations: {
+    notifications: 'Izaziso',
```

Comment:
This translation doesn't seem accurate based on reversing Google Translate.
Are you a native speaker of zulu language?